pkg/piecestore/psserver/psdb/psdb_test.go

```diff
@@ -12,13 +13,16 @@ import (
 	"time"
 
 	_ "github.com/mattn/go-sqlite3"
+	"github.com/stretchr/testify/require"
 
-	"storj.io/storj/internal/teststorj"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testidentity"
+	"storj.io/storj/pkg/bwagreement/testbwagreement"
+	"storj.io/storj/pkg/identity"
 	"storj.io/storj/pkg/pb"
-	"storj.io/storj/pkg/storj"
 )
 
-const concurrency = 10
+const concurrency = 1
```

Comment:
I'm pretty sure we don't want to test with concurrency of one.  If changing this was the only way to make the test work, there's probably something wrong.